Doc/library/tkinter.messagebox.rst

```diff
@@ -21,19 +22,169 @@ limited to:
 
    Create a default information message box.
 
+   The following options are supported:
+
+      *command*
+         Specifies the function to invoke when the user closes the dialog.
+         The name of the button clicked by the user to close the dialog is
+         passed as argument.
+         This is only available on macOS.
+
+      *default*
+         Gives the :ref:`symbolic name <messagebox-buttons>` of the default button
```

Comment:
```suggestion
         Gives the :ref:`symbolic name <messagebox-buttons>` of the button initially selected (highlighted)
```

Reply:
The Tk documentation uses term "selected" not as a synonym of "highlighted" but as "the one selected by the user". Most of documentation are copied from the Tk manpage with small adaptation, it is better to not introduce inconsistencies. The same terminology can be used in other parts of the Tk documentation (for example the default button is mentioned again three lines below).